src/neuroglancer/datasource/boss/api.ts

```diff
@@ -27,10 +28,20 @@ export type BossToken = string;
 export const credentialsKey = 'boss';
 
 export function fetchWithBossCredentials<T>(
-    credentialsProvider: CredentialsProvider<BossToken>, input: RequestInfo, init: RequestInit,
-    transformResponse: ResponseTransform<T>,
-    cancellationToken: CancellationToken = uncancelableToken): Promise<T> {
-  return fetchWithCredentials(
+  credentialsProvider: CredentialsProvider<BossToken>, input: RequestInfo, init: RequestInit,
+  transformResponse: ResponseTransform<T>,
+  cancellationToken: CancellationToken = uncancelableToken): Promise<T> {
+  return cancellableFetchOk(input, init, transformResponse, cancellationToken).catch((error) => {
+    // if (error.status === 504) {
+    //   // Gateway timeout can occur if the server takes too long to reply.  Retry.
+    //   return 'retry';
+    // }
+    if (error.status < 100) {

```

Comment:
for this line, try
```
    if (error.status !== 401 && error.status !== 403 && error.status !== 504) {
```
I am like 90% sure that'll work. I did see some weird behavior but I'm not sure how related it was to browser caching so please test it out too

Reply:
You were of course correct :) Added 500-codes to this list and it looks like it's running beautifully now :)